databases/week2/class2.sql

```diff
@@ -0,0 +1,64 @@
+-- Active: 1660903376490@@127.0.0.1@3306@hyf_lesson_1
```

Comment:
Very cool database! My only comment is that maybe in load table - you can do without loan_id, as you can create primary key with book_id and member_id combined

Reply:
yes thats good point